### PR TITLE
Update to latest snapshot.

### PR DESCRIPTION
**Pull request by anuraaga**

### CODE REVIEW

**Comment by anuraaga** [comment]
I built updated smoke test images locally and confirmed the pass - but I didn't push them since that would cause other PRs to fail, so this one will have failing smoke-tests

**Review by iNikem — APPROVED** [review]
Let's pray it will work

File: instrumentation/instrumentation.gradle [hunk 1]
```diff
@@ -75,21 +75,8 @@ shadowJar {
   relocate 'io.opentelemetry.instrumentation.api', 'io.opentelemetry.javaagent.shaded.instrumentation.api'
 
   // relocate OpenTelemetry API usage
-  relocate "io.opentelemetry.DefaultOpenTelemetry", "io.opentelemetry.javaagent.shaded.io.opentelemetry.DefaultOpenTelemetry"
-  relocate "io.opentelemetry.OpenTelemetry", "io.opentelemetry.javaagent.shaded.io.opentelemetry.OpenTelemetry"
-  relocate "io.opentelemetry.package-info", "io.opentelemetry.javaagent.shaded.io.opentelemetry.package-info"
-  relocate "io.opentelemetry.common", "io.opentelemetry.javaagent.shaded.io.opentelemetry.common"
-  relocate "io.opentelemetry.baggage", "io.opentelemetry.javaagent.shaded.io.opentelemetry.baggage"
+  relocate "io.opentelemetry.api", "io.opentelemetry.javaagent.shaded.io.opentelemetry.api"
```
Comment (mateuszrzeszutek) [inline instrumentation/instrumentation.gradle:78]: Nice!

File: javaagent-exporters/javaagent-exporters.gradle [hunk 2]
```diff
@@ -65,13 +65,13 @@ shadowJar {
...
+  relocate "io.opentelemetry.api.baggage", "io.opentelemetry.javaagent.shaded.io.opentelemetry.api.baggage"
   relocate "io.opentelemetry.context", "io.opentelemetry.javaagent.shaded.io.opentelemetry.context"
-  relocate "io.opentelemetry.internal", "io.opentelemetry.javaagent.shaded.io.opentelemetry.internal"
-  relocate "io.opentelemetry.metrics", "io.opentelemetry.javaagent.shaded.io.opentelemetry.metrics"
-  relocate "io.opentelemetry.spi", "io.opentelemetry.javaagent.shaded.io.opentelemetry.spi"
-  relocate "io.opentelemetry.trace", "io.opentelemetry.javaagent.shaded.io.opentelemetry.trace"
+  relocate "io.opentelemetry.api.internal", "io.opentelemetry.javaagent.shaded.io.opentelemetry.api.internal"
+  relocate "io.opentelemetry.api.metrics", "io.opentelemetry.javaagent.shaded.io.opentelemetry.api.metrics"
+  relocate "io.opentelemetry.api.spi", "io.opentelemetry.javaagent.shaded.io.opentelemetry.api.spi"
+  relocate "io.opentelemetry.api.trace", "io.opentelemetry.javaagent.shaded.io.opentelemetry.api.trace"
```
Comment (mateuszrzeszutek) [inline javaagent-exporters/javaagent-exporters.gradle:76]: Can you relocate just the api package?

```
relocate "io.opentelemetry.api", "io.opentelemetry.javaagent.shaded.io.opentelemetry.api"
```

**Review by mateuszrzeszutek — APPROVED** [review]

File: javaagent-exporters/javaagent-exporters.gradle (hunk 2 above)
Comment (anuraaga) [reply javaagent-exporters/javaagent-exporters.gradle]: Oops, missed this file thanks!

**Comment by anuraaga** [comment]
Confirmed my local smoke tests again after latest rebase

**Comment by trask** [comment]
FYI removed branch protection in order to merge this

**Pull request merged**